### ISSUE TITLE
Different balance in gwtc and waltonchain.pro

### ISSUE THREAD

**Issue by TkTzLabs** [disy-yin/waltonchain-gwtc#5]
Please see screenshot take at the same time.

- In GWTC web3.fromWei(wtc.getBalance(wtc.coinbase), "ether") and then wtc.coinbase
- wtc.coinbase address copied to waltonchain.pro

see screenshot balances are different in gwtc is 55 and in waltonchain.pro is 50

**Comment by TkTzLabs** [comment]
![gwtc-balances](https://user-images.githubusercontent.com/10755214/54864277-62673280-4d55-11e9-9558-4cef16e538e8.jpg)

**Comment by Angus23203** [comment]
Noticed this aswell, waltonchain.pro hasn't updated my balance for a couple of houres now

**Comment by BuggerSee** [comment]
This could be because the explorer is delayed a bit because its all cached. Did your status change yet?

**Comment by TkTzLabs** [comment]
Nope still 50 on waltonchain.pro ... in gwtc I alread mined one block

**Comment by BuggerSee** [comment]
Oh just noticed it too. Looks like the last block doesnt get added to the balance.

**Comment by TkTzLabs** [comment]
now its "last" two blocks

**Comment by disy-yin** [comment]
> Please see screenshot taken at the same time.
> 
> * In **GWTC web3.fromWei(wtc.getBalance(wtc.coinbase), "ether")** and then **wtc.coinbase**
> * **wtc.coinbase** address copied to **waltonchain.pro**
> 
> see screenshot balances are different in **gwtc is 55** and in **waltonchain.pro is 50**

Yes. 
To speed up access, the browser caches the data into the database.

**Comment by TkTzLabs** [comment]
> > Please see screenshot taken at the same time.
> > 
> > * In **GWTC web3.fromWei(wtc.getBalance(wtc.coinbase), "ether")** and then **wtc.coinbase**
> > * **wtc.coinbase** address copied to **waltonchain.pro**
> > 
> > see screenshot balances are different in **gwtc is 55** and in **waltonchain.pro is 50**
> 
> Yes.
> To speed up access, the browser caches the data into the database.

Hmmmhmm, just tried also in anonymous mode of chrome (sandbox) -> there arent any data cached yet and result it is same.

**Comment by CyberJimbo** [comment]
yep, same problem for me, switched to another browser in private mode and dont see account balance

**Comment by disy-yin** [comment]
> yep, same problem for me, switched to another browser in private mode and dont see account balance

https://waltonchain.pro
This browser is used with gwtc only.

**Comment by TkTzLabs** [comment]
yes, in gwtc right now I have 70 coins mined and on waltonchain.pro I have still 50. You can check that out by yourself : 0xd21f7069292d30c0dab325fa9ebde352a2968c37

**Comment by AboldUSER** [comment]
Looks like a big lag for me too. gwtc shows me with 85 while waltonchain.pro shows me with 55

**Comment by disy-yin** [comment]
> yes, in gwtc right now I have 70 coins mined and on waltonchain.pro I have still 50. You can check that out by yourself : 0xd21f7069292d30c0dab325fa9ebde352a2968c37

OK. Thanks.
We will follow this issue.

**Comment by TkTzLabs** [comment]
Starting another round of tests from scratch so I will keep you in the loop with update.

**Comment by TkTzLabs** [comment]
Very interesting point is that Balance is in gwtc and on waltonchain.pro it is not but there is information about mined block(s)

![wtc_issue](https://user-images.githubusercontent.com/10755214/54879692-370b4300-4e3c-11e9-8380-833005f82361.jpg)

**Comment by disy-yin** [comment]
The data has been updated and synchronized.

**Comment by Angus23203** [comment]
waltonchain.pro has been down for days now btw. No data on it
